ArduPlane/Plane.cpp

```diff
@@ -280,6 +280,11 @@ void Plane::update_logging10(void)
         camera_mount.write_log();
     }
 #endif
+#if AP_RANGEFINDER_ENABLED
+    if (should_log(MASK_LOG_NTUN)) {
```

Comment:
Should we check that rangefinder at least could be used in this mode? Maybe just check `rangefinder_landing > 0` and there is a enabled rangefinder for the set direction.

Reply:
As far as I can see, whenever `AP_RANGEFINDER_ENABLED`, the task `read_rangefinder` will call `rangefinder_height_update()` always, even if there is no instantiated rangefinder backend.

Within there, a `range_ok` check is built from the first valid rangefinder. If there is none, then
```
        rangefinder_state.in_range_count = 0;
        rangefinder_state.in_range = false;
```
which I still think is worth logging.

Do you agree?

Reply:
Any reason not to also check `rangefinder_landing > 0` ?

Reply:
Right, but is should never be used if `rangefinder_landing` is not set. Although I guess you may want to do some flights to see how the rangefinder is getting on before you start using it.

Reply:
@IamPete1 could you approve?